wp-includes/class-wp-service-worker-scripts.php

```diff
@@ -181,7 +181,7 @@ public function do_item( $handle, $group = false ) {
 	/**
 	 * Get validated path to file.
 	 *
-	 * @param string $url Relative path.
+	 * @param string $url Relative path or false.
```

Comment:
This change can be reverted now that the `is_string()` check is being used to verify that `$header_image` is _not_ false, no?

```suggestion
	 * @param string $url Relative path.
```

Reply:
Done :)